javascript/javascript3/week1/index.js

```diff
@@ -25,4 +25,69 @@
   // Along with above URL one need to use parameters like cityname, zip code, geolocation and api key as well
   // one example of using this api with different parameters is following 
   //'https://api.openweathermap.org/data/2.5/weather?q=London&units=metric&appid=2e67673623b242fce63358a81c88036e'
-  
\ No newline at end of file
+  
+// !--Weather app--!
+
+const cityName = document.getElementById("heading")
+      const temperature = document.getElementById("temppara")
+      const feeltemp = document.getElementById("feelslike")
+      const wind = document.getElementById("cl-1")
+      const description = document.getElementById("cl-2")
+      const riseTime = document.getElementById("cl-3")
+      const sunTime = document.getElementById("cl-4")
+      
+      const myButton = document.getElementById("location")
```

Comment:
All of these variables use the same `document.getElementById("something")` right? Why not use a function that would return the value? Would make the code nicer and reusable.